donkeycar/parts/keras.py

```diff
@@ -158,8 +158,7 @@ def __init__(self, num_outputs=2, input_shape=(120, 160, 3), roi_crop=(0, 0), *a
         self.compile()
 
     def compile(self):
-        self.model.compile(optimizer=self.optimizer,
-                loss='mse')
+        self.model.compile(optimizer=self.optimizer, loss='mse')
```

Comment:
Does this needs to be in this PR?
Does it affect a new datastore format directly?

I believe it suits better as a new PR focused on optimizing the AI/ML part, so maybe it is better to add it to the dev branch as separate PR, especially if it is not related to the datastore itself, what do you think?

Reply:
This is useful because it shows how to add a model to the new pipeline.

Reply:
OK this is just a newline remove.